python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -619,6 +650,19 @@ def terminate_scheduled_nodes(self):
         # explicitly disabled.
         if self.worker_rpc_drain:
             self.drain_nodes_via_gcs(self.nodes_to_terminate)
+        # Clean up multi-host replicas to delete
+        for node in self.nodes_to_terminate:
```

Comment:
TODO: Will `replicas_to_nodes` and `replicas_to_delete` be called anywhere after the function call of `terminate_scheduled_nodes`?